<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>629</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>624</width>
    <height>629</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>624</width>
    <height>629</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Object Generator</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>341</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QCheckBox" name="checkBox_autoRotation">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true">For sprites that should rotate with the camera view. Note that these are fake rotations based on mirroring</string>
     </property>
     <property name="text">
      <string>Auto-Rotation</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_suffix">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true">Add suffix here</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prefix">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true">The name your object will appear ingame. If you do not tick Auto-Naming every object will be called the same</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_objectIngameName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Object ingame name</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_autoNaming">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>230</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable Auto-Naming. Then you can add a global pre- and suffix to your object, every object gets the respective template name added in between. The case 'Fulltile' here is such an example.&lt;/p&gt;&lt;p&gt;If not enabled, then all generated objects will get the &lt;span style=&quot; font-style:italic;&quot;&gt;same&lt;/span&gt; given name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Auto-Naming</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_version">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true">Unless you don't revise an already released object, keep this at 1.0</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_version">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Version</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_objectID">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Object ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_objectID">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Unique object identifier - a suffix will be added for each generated object type&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_authorID">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true">Your identity/name for the object ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_authorID">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Author ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_author">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Author</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_author">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true">Your name, for collaborations separate by comma</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_primaryRemap">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Use Primary Remap</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_secondaryRemap">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Use Secondary Remap</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_loadImageButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Load your base path texture. Sprite should have shape 64px*31px , else it will be cropped from the top left corner</string>
     </property>
     <property name="text">
      <string>Load Base Image</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prefix">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>260</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Prefix</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_suffix">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Suffix</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_displayName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>326</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {

	background-color: rgb(230,230, 230);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Display name</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_generate">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Objects</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_outputFolder">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_changeOutputFolder">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="sprite_view">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>172</width>
      <height>132</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {

	background-color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_OutputFolder">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Folder</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_templateList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>341</width>
      <height>191</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_selectAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clearAll">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>580</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_importTemplate">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>580</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Template</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_GenerateReturn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton_loadImageButton</tabstop>
  <tabstop>checkBox_primaryRemap</tabstop>
  <tabstop>checkBox_autoRotation</tabstop>
  <tabstop>checkBox_secondaryRemap</tabstop>
  <tabstop>lineEdit_author</tabstop>
  <tabstop>lineEdit_authorID</tabstop>
  <tabstop>lineEdit_objectID</tabstop>
  <tabstop>doubleSpinBox_version</tabstop>
  <tabstop>checkBox_autoNaming</tabstop>
  <tabstop>lineEdit_prefix</tabstop>
  <tabstop>lineEdit_suffix</tabstop>
  <tabstop>pushButton_generate</tabstop>
  <tabstop>lineEdit_outputFolder</tabstop>
  <tabstop>pushButton_changeOutputFolder</tabstop>
  <tabstop>listWidget_templateList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
